containers/ecr-viewer/.env.sample

```diff
@@ -9,18 +9,27 @@ AZURE_CONTAINER_NAME=ecr-viewer-files
 
 # AWS Connection
 AWS_REGION=us-east-1
-AWS_CUSTOM_ENDPOINT=http://localhost:4566
 ECR_BUCKET_NAME=ecr-viewer-files
 AWS_ACCESS_KEY_ID=123
 AWS_SECRET_ACCESS_KEY=xyz
+# Only for local use
+AWS_CUSTOM_ENDPOINT=http://localhost:4566
+
+# GCP GCS
+GCS_BUCKET_NAME=ecr-viewer-files
```

Comment:
can we use `ECR_BUCKET_NAME` here instead of a net new env var?

Reply:
I'll open another PR to replace those two!